packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  SpanExporter,
+  ReadableSpan,
+  ExportResult,
+} from '@opentelemetry/basic-tracer';
+import * as jaegerTypes from './types';
+import { ConsoleLogger } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
+import { spanToThrift } from './transform';
+
+/**
+ * Format and sends span information to Jaeger Exporter.
+ */
+export class JaegerExporter implements SpanExporter {
+  private readonly _logger: types.Logger;
+  private readonly _process: jaegerTypes.ThriftProcess;
+  private _sender: typeof jaegerTypes.UDPSender;
+
+  constructor(config: jaegerTypes.ExporterConfig) {
+    this._logger = config.logger || new ConsoleLogger();
```

Comment:
I think `new ConsoleLogger()` should be `new NoopLogger()`. The reason is that we can't pass a logLevel and config accept a logger. OT should not display logs by default and I think that default logLevel of `ConsoleLogger()` is debug, right ? WDYT ?

Reply:
Changed to `NoopLogger`.